CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.2.1(upcoming release)
+
+### Fixes 
```

Comment:
I would put it as an enhancement, since it was not a bug

Reply:
done